### PR TITLE
app-admin/aerospike-amc-community: various changes (see first comment)

### PR DESCRIPTION
**Pull request by scardracs**
Bump to new version
Removed inherits
Fixed GLEP 81
Added acct-group/user

Package-Manager: Portage-3.0.18, Repoman-3.0.3
Acked-by: Patrick Lauer patrick@gentoo.org
Signed-off-by: Marco Scardovi marco@scardovi.com

### CODE REVIEW

**Comment by scardracs** [comment]
It's based on #20417 but the old branch was a mess so I've make a new one

File: app-admin/aerospike-amc-community/aerospike-amc-community-5.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND="
+	acct-group/aerospike-amc-community
+	acct-user/aerospike-amc-community
+"
+
+RDEPEND="${DEPEND}"
```
Comment (ezzieyguywuf) [inline app-admin/aerospike-amc-community/aerospike-amc-community-5.0.0.ebuild:21]: Is there a good reason not to just use `RDEPEND` directly? Is the user and group needed during installation?

```suggestion
RDEPEND="
	acct-group/aerospike-amc-community
	acct-user/aerospike-amc-community
"
```

**Review by ezzieyguywuf — APPROVED** [review]
Other than the one comment I left below, I would also swap the two commits so that you create the user and group on commit 1, and then add the ebuild on commit 2 (or just squash it all together...)

File: app-admin/aerospike-amc-community/aerospike-amc-community-5.0.0.ebuild (hunk 1 above)
Comment (scardracs) [reply app-admin/aerospike-amc-community/aerospike-amc-community-5.0.0.ebuild:21]: Oh, i missed it, thanks

File: acct-user/aerospike-amc-community/aerospike-amc-community-0.ebuild [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="Aerospike-amc-community acct-user"
+
+ACCT_USER_ID=96
+
+ACCT_USER_GROUPS=( "${PN}" )
```
Comment (ConiKost) [inline acct-user/aerospike-amc-community/aerospike-amc-community-0.ebuild:12]: Please write clear text, without any variables.

File: app-admin/aerospike-amc-community/aerospike-amc-community-4.0.19-r1.ebuild [hunk 3]
```diff
@@ -1,24 +1,25 @@
 # Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
-inherit user
+EAPI=7
```
Comment (ConiKost) [inline app-admin/aerospike-amc-community/aerospike-amc-community-4.0.19-r1.ebuild:4]: Please bump revision, if bumping EAPI.

File: app-admin/aerospike-amc-community/aerospike-amc-community-4.0.19-r1.ebuild [hunk 4]
```diff
@@ -1,24 +1,25 @@
...
 
-RDEPEND=""
-DEPEND="${RDEPEND}"
+RDEPEND="
+	acct-group/aerospike-amc-community
+	acct-user/aerospike-amc-community
+"
 
-S=${WORKDIR}
+S="${WORKDIR}"
```
Comment (ConiKost) [inline app-admin/aerospike-amc-community/aerospike-amc-community-4.0.19-r1.ebuild:19]: S comes after SRC_URI.

File: app-admin/aerospike-amc-community/aerospike-amc-community-5.0.0.ebuild [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	acct-group/aerospike-amc-community
+	acct-user/aerospike-amc-community
+"
+
+S="${WORKDIR}"
```
Comment (ConiKost) [inline app-admin/aerospike-amc-community/aerospike-amc-community-5.0.0.ebuild:21]: S comes afteR SR_URI.

File: acct-user/aerospike-amc-community/aerospike-amc-community-0.ebuild [hunk 6]
```diff
@@ -0,0 +1,14 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="Aerospike-amc-community acct-user"
+
+ACCT_USER_ID=96
```
Comment (ConiKost) [inline acct-user/aerospike-amc-community/aerospike-amc-community-0.ebuild:10]: New users/groups should used a higher uid/gid. See `used_free_uidgids.sh`.
`Recommended UID+GID pair: 211`

File: acct-group/aerospike-amc-community/aerospike-amc-community-0.ebuild [hunk 7]
```diff
@@ -0,0 +1,9 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-group
+
+DESCRIPTION="Aerospike-amc-community acct-group"
+ACCT_GROUP_ID=96
```
Comment (ConiKost) [inline acct-group/aerospike-amc-community/aerospike-amc-community-0.ebuild:9]: New users/groups should used a higher uid/gid. See `used_free_uidgids.sh`.
`Recommended UID+GID pair: 211`

File: acct-user/aerospike-amc-community/aerospike-amc-community-0.ebuild (hunk 6 above)
Comment (ConiKost) [reply acct-user/aerospike-amc-community/aerospike-amc-community-0.ebuild:10]: Also please will a PR at https://github.com/gentoo/api-gentoo-org/tree/master/files

File: acct-group/aerospike-amc-community/aerospike-amc-community-0.ebuild (hunk 7 above)
Comment (ConiKost) [reply acct-group/aerospike-amc-community/aerospike-amc-community-0.ebuild:9]: Also please will a PR at https://github.com/gentoo/api-gentoo-org/tree/master/files

**Review by ConiKost — CHANGES_REQUESTED** [review]

**Pull request closed without merging**